designer/client/src/components/toolbars/activities/ActivitiesPanelRow.tsx

```diff
@@ -27,7 +27,8 @@ export const ActivitiesPanelRow = memo(({ index, style, setRowHeight, handleShow
         () => activities.findIndex((activeItem) => activeItem.uiType === "item" && activeItem.type === "SCENARIO_DEPLOYED"),
         [activities],
     );
-    const isRunning = firstDeployedIndex === index && scenarioState.status.name === "RUNNING";
+    const scenarioStatusesToActiveDeploy = ["RUNNING", "SCHEDULED"];
+    const isRunning = firstDeployedIndex === index && scenarioStatusesToActiveDeploy.includes(scenarioState.status.name);
```

Comment:
Let's call this variable `isActive`

Reply:
Good point, i change it to isDeploymentActive